src/main/resources/db/migration/V1__init.sql

```diff
@@ -1,5 +1,5 @@
 create table users(
-    id bigint primary key unique auto_increment,
+    id bigint auto_increment,
```

Comment:
why remove them?

Reply:
When we use **id bigint primary key unique auto_increment** it's not going to work H2.